app/views/content/non-uk-teachers/fees-and-funding-for-non-uk-trainees.md

```diff
@@ -269,10 +192,8 @@ Talk to your training provider about:
 * home fee rate
 * a [bursary or scholarship](/funding-and-support/scholarships-and-bursaries)
 
-## Get help with financial support
+## Get help
```

Comment:
Just wondered why you removed the financial support? Do they offer other types of support too?

Reply:
Mainly because I'd replaced financial support with student finance (now loans) but also because I don't want to give the impression they can GET financial support, if you see what I mean, as they probably can't...